backend/peak/mongoUtils.py

```diff
@@ -0,0 +1,34 @@
+from pymongo.mongo_client import MongoClient
+# from pymongo.server_api import ServerApi
+
+
```

Comment:
@jrojewski, create a task to make a refactor later, please. We can keep it as it is right now.